libs/island-ui/core/src/lib/CategoryCard/CategoryCard.tsx

```diff
@@ -107,6 +108,7 @@ const Component = forwardRef<
       hyphenate = false,
       tagOptions,
       autoStack,
+      children,
```

Comment:
This card does not have children in the design and can brake the overall design of the card. Is this accepted by a designer in the team?

Reply:
Removed. We don't have all the information that we need for the CurrentSchool component and so we will not be making changes to CategoryCard